app/src/main/java/com/iic/mokojin/activities/PlayerQueueFragment.java

```diff
@@ -68,11 +68,15 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
             }
         });
         mQueueListView.enableSwipeToDismiss();
-        scheduleUpdateClock();
 
         return rootView;
     }
 
+    @Override public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        scheduleUpdateClock();
```

Comment:
Ok


> On 16 במרץ 2015, at 13:28, udiWertheimer <notifications@github.com> wrote:
> 
> In app/src/main/java/com/iic/mokojin/activities/PlayerQueueFragment.java:
> 
> >  
> >          return rootView;
> >      }
> >  
> > +    @Override public void onActivityCreated(Bundle savedInstanceState) {
> > +        super.onActivityCreated(savedInstanceState);
> > +        scheduleUpdateClock();
> @yonbergman I moved this here after @giladgo 's recommendation, because it was called before the activity was ready when opened through the push notification. Speak now or forever hold your peace..
> 
> —
> Reply to this email directly or view it on GitHub.
>